infrastructures/infrastructure-vmware/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/VMWareInfrastructure.java

```diff
@@ -99,6 +99,9 @@
     @Configurable(description = "Command used to download the worker jar")
     protected String downloadCommand = generateDefaultDownloadCommand();
 
+    @Configurable(description = "Optional list of MAC addresses separated by comma ',' to assign on new cloned VMs (1 per instance)")
```

Comment:
Removed in https://github.com/vinseon/nodesource-addons/commit/ac0f9823268618d9e9dd5b1a20c4559b94364591